### ISSUE TITLE
No longer detecting MacOS in version 1.1.1

### ISSUE THREAD

**Issue by webappaloosa** [charlesguse/run-script-os#30]
### Version 1.1.1 is not detecting MacOS

MacOS (v10.15.5).

-----------------------------
### package.json:
-----------------------------
```
 {
  "name": "ngx-project",
  "version": "0.0.1",
  "scripts": {
    "ostest": "run-script-os",
    "ostest:macos": "echo 'This will run on MacOS'",
    "ostest:windows": "echo 'This will run on Windows'",
    "ostest:default": "echo 'This will run on any platform that does not have its own script'"
  },
  "dependencies": {
    "@angular/animations": "~10.0.9",
    "@angular/cdk": "^10.1.3",
    "@angular/common": "~10.0.9",
    "@angular/compiler": "~10.0.9",
    "@angular/core": "~10.0.9",
    "@angular/flex-layout": "^10.0.0-beta.32",
    "@angular/forms": "~10.0.9",
    "@angular/material": "^10.1.3",
    "@angular/platform-browser": "~10.0.9",
    "@angular/platform-browser-dynamic": "~10.0.9",
    "@angular/router": "~10.0.9",
    "rxjs": "~6.5.5",
    "tslib": "^2.0.0",
    "zone.js": "~0.10.3"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.1000.6",
    "@angular/cli": "~10.0.6",
    "@angular/compiler-cli": "~10.0.9",
    "@types/jasmine": "~3.5.0",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "^12.11.1",
    "codelyzer": "^6.0.0",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~5.0.0",
    "karma": "~5.0.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~3.0.2",
    "karma-jasmine": "~3.3.0",
    "karma-jasmine-html-reporter": "^1.5.0",
    "npm-run-all": "^4.1.5",
    "protractor": "~7.0.0",
    "run-script-os": "^1.1.1",
    "ts-node": "~8.3.0",
    "tslint": "~6.1.0",
    "typescript": "~3.9.5"
  }
```

-----------------------------
### terminal output
-----------------------------
> ComputerName:dev abcdefg$ npm run ostest
>
> ngx-project@0.0.1 ostest /Users/abcdefg/projects/ngx-project/dev
> run-script-os
>
>
> ngx-project@0.0.1 ostest:default /Users/abcdefg/projects/ngx-project/dev
> echo 'This will run on any platform that does not have its own script'
>
>This will run on any platform that does not have its own script

**Comment by webappaloosa** [comment]
I rolled back to run-script-os@1.0.7, and it is working correctly.

**Comment by shaaraddalvi** [comment]
Yes, seems like the alias `macos` for `darwin` is not present. Switching to `darwin` did the trick for me.

**Comment by charlesguse** [comment]
This has since been fixed in 1.1.4 and tests have been added to prevent this from happening in the future. Sorry about the inconvenience!
